### DISCUSSION TITLE
Blazor webassembly in decoupled multi tenant

### DISCUSSION THREAD

**Discussion by mattobox** [OrchardCMS/OrchardCore#12007] [Q&A] [answered]
I was able to get blazor webassembly to work in Orchard Core using the method described here:

[Blazor Webassembly Integrated with Orchard Core CMS](https://youtu.be/Zu4D_JMcCTY) 

It works fine when there is the single default tenant but I haven’t been able to get it to work to support multiple tenants. Any guidance on getting blazor webassembly to work like this?

**Comment by ns8482e** [comment, marked as answer]
- Create blazor web assembly client
- Create OC module and add reference of blazor web assembly client 
- Remove UseStaticFiles from host 
- Add UseBlazorFrameworkFiles and Add UseStaticFiles in OC module’s startup

**Reply by mattobox** [reply]
Thanks so much. Just the head start I needed. Cheers!

**Reply by mattobox** [reply]
Do you offer any consulting services for Orchard Core projects?

**Reply by ns8482e** [reply]
@mattobox yes you can IM @ns8482e on gitter
